packages/zipkin/src/batch-recorder.js

```diff
@@ -1,11 +1,13 @@
-const {now} = require('./time');
-const thriftTypes = require('./gen-nodejs/zipkinCore_types');
-const {
-  MutableSpan,
-  Endpoint,
-  ZipkinAnnotation,
-  BinaryAnnotation
-} = require('./internalRepresentations');
+const {now, hrtime} = require('./time');
+const {Span, Endpoint} = require('./model');
+
+function PartialSpan(traceId) {
```

Comment:
ended up aligning name with partial span which is used elsewhere in the same file